composer.json

```diff
@@ -7,7 +7,7 @@
 	"authors": [
 		{
 			"name": "Jaspaul Bola",
-			"email": "jaspaul.b@gamil.com",
+			"email": "jaspaul.b@gmail.com",
```

Comment:
Good catch. I think you should probably add yourself as a contributor here as well.